pynsee/macrodata/_get_dataset_metadata.py

```diff
@@ -112,3 +52,5 @@ def _get_dataset_metadata(dataset, update=False):
         idbank_list_dataset = idbank_list_dataset.reset_index(drop=True)
 
     return idbank_list_dataset
+        
+    
```

Comment:
remove

Reply:
b27d6751ff1dc1738b522be9dce0680fd43e41d7 should tackle the issue